packages/zipkin-instrumentation-restify/src/restifyMiddleware.js

```diff
@@ -36,12 +34,7 @@ module.exports = function restifyMiddleware({tracer, serviceName, port = 0}) {
         res.removeListener('finish', onCloseOrFinish);
 
         tracer.scoped(() => {
-          const error =
-            res.statusCode !== 0 && (res.statusCode < LOWER_LIMIT || res.statusCode > UPPER_LIMIT)
-            ? res.statusCode
-            : null;
-
-          instrumentation.recordResponse(id, res.statusCode, error);
+          instrumentation.recordResponse(id, res.statusCode);
```

Comment:
`restify` used to add the error code based on status code but now we moved the logic to the http server instrumentation.